hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -194,6 +194,37 @@
   public static final String EXTERNAL_RECORD_AND_SCHEMA_TRANSFORMATION = AVRO_SCHEMA + ".externalTransformation";
   public static final String DEFAULT_EXTERNAL_RECORD_AND_SCHEMA_TRANSFORMATION = "false";
 
+  public static final String PRE_COMMIT_VALIDATORS = "hoodie.precommit.validators";
```

Comment:
can we create a new Config class for this, instead of overloading the WriteConfig?

Reply:
Added